girder/utility/model_importer.py

```diff
@@ -27,6 +27,14 @@
 # cache of instantiated models that have been lazy-loaded.
 _modelInstances = {}
 
+_pluginModelClasses = {}
+
+
+def girder_model(cls, name, plugin=None):
+    plugin_registry = _pluginModelClasses.setdefault(plugin, {})
+    plugin_registry[name] = cls
+    logger.info('Registered model %s in plugin %s' % (name, plugin))
```

Comment:
This is just a temporary hack to register plugin models for `ModelImporter`.  Eventually, I want to create a metaclass or a class decorator to register models and make it consistent between core and plugin models.